configs/HPECray-EPYC3-MI250X-Slingshot/system_definition.yaml

```diff
@@ -0,0 +1,28 @@
+# Copyright 2023 Lawrence Livermore National Security, LLC and other
+# Benchpark Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: Apache-2.0
+
+system_definition:
+  name: HPECray-EPYC3-MI250X-Slingshot
+  manufacturer:
+    vendor: HPE-Cray
+    name: EX235a
+  processor:
+    vendor: AMD
+    name: EPYC-Zen3
+    ISA: x86_64
+  accelerator:
+    vendor: AMD
+    name: MI250X
+    GPU_ISA: gfx90a
+  interconnect:
+    vendor: HPE-Cray
+    name: Slingshot-11
+  OS:
+    name: HPE-Cray-OS
```

Comment:
This is not accurate, we ran on TOSS. There's nothing in the config that makes it arch specific though.

Reply:
The intent here is the "normal" machine definition others recognize, and that list is mostly what you see on the top500.  Technically even tioga came with Cray OS; I didn't mention the OS in the "system-tested".

Do you think we should drop the OS info?  I think it is nice to have it for completeness.  And, maybe we need to indicate the full tested configuration.  While it is nice to have a short system-tested description here, I do worry that it is coupled with the hardware description - where what we really need is the full spec of what was tested, and the date it was tested - because if i tested on tioga a year ago but have since updated these scripts and the machine, this info is obsolete/useless/misleading?

Reply:
I thought the idea was that these would be continually tested, so listing the system it is tested on on an ongoing basis would still be meaningful?

Reply:
I would be fine with dropping the `OS` information -- it's really only relevant to specs that we use from externals, and those are also recorded. Since different sites can install different OSs on the same system or different software from the OS, leaving it out might be better

Reply:
Continually tested: yes, that would be ideal, and hopefully we figure out a way to pull in "last tested on" from the CI.

Ok, I will remove the OS.